server.lua

```diff
@@ -1,6 +1,6 @@
 local QBCore = exports['qb-core']:GetCoreObject()
 
-QBCore.Functions.CreateUseableItem("radio", function(source, item)
+QBCore.Functions.CreateUseableItem("radio", function(source, _)
```

Comment:
You can remove this underscore completely

Reply:
This one makes sense.

Reply:
Not just this one, all the other ones make sense as well because it's the same context, when a variable is not used and there is nothing else after that that's being used, you can remove it including everything after it, lua won't do anything with it then and only send the variable you declared

Reply:
```suggestion
QBCore.Functions.CreateUseableItem("radio", function(source)
```